src/main/java/ca/xqz/tweetmouth/TweetLoader.java

```diff
@@ -59,6 +63,30 @@ public void loadTweets(List<Tweet> tweets) {
         }
     }
 
+    private void maybeCreateIndex() {
+        IndicesExistsResponse resp = client.admin().indices().exists(new IndicesExistsRequest(index)).actionGet();
+        if (!resp.isExists()) {
+            System.out.println("Index doesn't exist; creating it");
```

Comment:
Can you separate the IndicesExistsResponse and request into a separate `bool indexExists()` function and have a separate `void createIndex()` function?